src/App.js

```diff
@@ -2,8 +2,8 @@ import * as React from 'react';
 import 'react-notifications/lib/notifications.css';
 import { hot } from 'react-hot-loader';
 import { BrowserRouter as Router, Route, Redirect } from 'react-router-dom';
-import CssBaseline from '@material-ui/core/CssBaseline';
-import { ThemeProvider } from '@material-ui/core/styles';
+import CssBaseline from '@mui/material/CssBaseline';
+import { ThemeProvider, StyledEngineProvider } from '@mui/material/styles';
```

Comment:
Why do we need StyledEngineProvider?

Reply:
It gives custom styles precedence over your MUI